ivy/functional/frontends/numpy/creation_routines/building_matrices.py

```diff
@@ -36,3 +36,8 @@ def vander(x, N=None, increasing=False):
     elif ivy.is_bool_dtype or ivy.is_int_dtype(x):
         x = x.astype(ivy.int64)
     return ivy.vander(x, N=N, increasing=increasing)
+
+
+# diagflat
+def diagflat(v, k=0, offset=0,  padding_value=0, align="RIGHT_LEFT", num_rows=-1, num_cols=-1):
```

Comment:
I think this is flipped, numpy only has `v` and `k`, it's ivy which has more arguments